dfds_ds_toolbox/analysis/plotting.py

```diff
@@ -144,45 +143,25 @@ def get_trend_stats(
 
 
 def plot_regression_predicted_vs_actual(
```

Comment:
Should we have a unit test for this one?

Reply:
Added unit test